docs/api/schemas/concept/embedded.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$id": "#/skos:Concept/embedded",
+    "type": "object",
+    "title": "embedded skos:Concept",
+    "required": [
+        "@id",
+        "@type",
+        "skos:prefLabel"
+    ],
+    "properties": {
+        "@id": {
+            "$id": "#/skos:Concept/embedded/@id",
+            "type": "string",
+            "title": "A dereferencable URI to a Concept Scheme",
+            "required": true,
+            "examples": [
+                "http://taxman.com/0.1/concept-scheme/1"
+            ],
+            "pattern": "^(.*)$"
+        },
+        "@type": {
+            "$id": "#/skos:Concept/embedded/@type",
+            "const": "skos:Concept"
+        },
+        "skos:prefLabel": {
```

Comment:
Looks like a lot of the fields are missing here, as well as in the other concept schemas

Reply:
This is the embedded schema (i.e. used to present a list of concepts), which is why it only has preferred label, and ID.

Reply:
Fair enough on this one, but the same thing applies to `request.json` and `response.json` - is that deliberate?

Reply:
Nope, I'm updating those with all the properties.